nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/Inserted.java

```diff
@@ -0,0 +1,60 @@
+package no.unit.nva.model.associatedartifacts.file;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.time.Instant;
+import java.util.Objects;
+import no.unit.nva.model.Username;
+import nva.commons.core.JacocoGenerated;
+
+@JsonTypeInfo(use = Id.NAME, property = "type")
+@JsonTypeName(PublishedFile.TYPE)
+public class Inserted {
+
+    @JsonProperty("insertedBy")
+    private final Username insertedBy;
+
+    @JsonProperty("insertedDate")
+    private final Instant insertedDate;
+
```

Comment:
Considered creating a modifiedDate field, but then we are getting very close to creating a audit trail and I am not sure if we want to do that without planning.

Reply:
Possibly `uploaded` here too?